app/controllers/flood_risk_engine/application_controller.rb

```diff
@@ -2,6 +2,8 @@ module FloodRiskEngine
   class ApplicationController < ActionController::Base
     protect_from_forgery with: :exception
 
+    layout ->(_) { 'application' }
+
```

Comment:
Just wondering - should the engine be specifying the layout the host app will use? Just compared to devise and its agnostic about layout